pages/index.html

```diff
@@ -4,9 +4,41 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Milk Punchies Home Page</title>
+  <link rel="stylesheet" href="../styles/style.css">
+  <link rel="stylesheet" href="../styles/index.css">
 </head>
+
 <body>
+  <div class="container">
+    <nav>
+        <a class="nav-target" href="index.html">Home Page</a>
+        <a class="nav-target" href="facts.html">Fun Facts!</a> 
+        <a class="nav-target" href="gallery.html">Gallery Page</a>
+    </nav>
+    
   
+    <main class="content-container">
+      <h1 class="page-header">Milk Punch: The Forgotten Classic Cocktail</h1>
+      <section>
+        <p>
+          Picture this: milk and alcohol. 
+          You heard that right! Not to worry my lactase-deficient friends. 
+          Milk punch is a delightful cocktail artisanally made from milk and spices, magically reduced down into a wonderully clear alcocholic beverage. 
+          Learn more about it on this website. 
+        </p>
+      </section>
+    </main>
+  
+    <footer>
+      <div class="links-container">
+        <a href="https://www.thecarlile.com/menu/cocktail-culture/">Find Milk Punch in Seattle</a>
+        <a href="https://www.bittersandbottles.com/products/rockeys-milk-punch">Buy Milk Punch Online</a>
+        <a href="https://www.youtube.com/watch?v=X17NxjRPeb4&ab_channel=TheWeePearl">Make Your Own Milk Punch</a>
+      </div>
+      <h4>&copy; The Milk Punch Website Copyright 2023</h4>
```

Comment:
Yay [HTML character entities](https://www.w3schools.com/html/html_entities.asp)! Glad to see y'all use it